1650961433782	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1650961434067	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile 304
1650961763489	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1650961763489	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1650961763492	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1650961763493	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1650961763549	Sync.SyncScheduler	DEBUG	Woke from sleep.
1650961763549	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1650961763549	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1650961763549	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1650961765550	Sync.Service	DEBUG	User-Agent: Firefox/99.0.1 (Linux x86_64) FxSync/1.101.0.20220413030647.desktop
1650961765550	Sync.Service	INFO	Starting sync at 2022-04-26 13:59:25 in browser session FGmVsTalqJKg
1650961765550	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1650961765552	Sync.Status	INFO	Resetting Status.
1650961765552	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1650961765553	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1650961765553	FirefoxAccounts	DEBUG	getOAuthToken enter
1650961765553	FirefoxAccounts	TRACE	getCachedToken returning cached token
1650961765553	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1650961765553	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1650961765557	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1650961765557	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 488\"  data: no]"}
1650961765558	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/194602547/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1650961765558	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1650961765558	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1650961765558	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1650961765558	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1650961765561	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
